### PR TITLE
AArch64: Stub files of InstOpCode

### PR DESCRIPTION
**Pull request by knn-k**
This commit implements stubs of InstOpCode files for aarch64.

Signed-off-by: knn-k <konno@jp.ibm.com>

### CODE REVIEW

**Comment by knn-k** [comment]
Minimum stubs only.  InstOpCode.hpp is needed in implementing OMRLinkage.hpp.

File: compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
```
Comment (0xdaryl) [inline compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp]: This file needs a copyright header as well, even though it is an incomplete C++ file.

File: compiler/aarch64/codegen/InstOpCode.hpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef TR_ARM64_INSTOPCODE_INCL
+#define TR_ARM64_INSTOPCODE_INCL
+
+#include "codegen/OMRInstOpCode.hpp"
+
+#ifndef OPS_MAX
+#define OPS_MAX    TR::InstOpCode::ARM64NumOpCodes
```
Comment (0xdaryl) [inline compiler/aarch64/codegen/InstOpCode.hpp]: What will this be used for?  Nothing in the current OMR code generators (nor OpenJ9 for that matter) uses this macro even on P and Z which also define it.  I'd rather we don't define this until we actually have a use for it.

Comment (knn-k) [reply compiler/aarch64/codegen/InstOpCode.hpp]: This macro came from the p version of InstOpCode.  I am removing it.

**Comment by knn-k** [comment]
Updated the files reflecting review comments.

**Pull request merged**